### DISCUSSION TITLE
Bitmap pixel font is unsupported.

### DISCUSSION THREAD

**Discussion by jgedri** [zed-industries/zed#14087] [Feature Requests]
I'm trying to build Zed on Windows 10 version 10.0.19045 and getting the following error. `[2024-07-10T05:14:09-04:00 ERROR] The bitmap pixel format is unsupported. (0x88982F80) ` What does it mean, and how can I fix it?

**Comment by xtrasmal** [comment]
Check the blog where Zed's font choices are explained.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
